css/css-color/parsing/color-invalid-relative-color.html

```diff
@@ -28,9 +28,9 @@
     test_invalid_value(`color`, `rgb(from rebeccapurple l g b)`);
     test_invalid_value(`color`, `rgb(from rebeccapurple h g b)`);
 
-    // Testing invalid function name variation (only rgb() is valid, rgba() is invalid)
-    test_invalid_value(`color`, `rgba(from rebeccapurple r g b)`);
-    test_invalid_value(`color`, `rgba(from rgb(10%, 20%, 30%, 40%) r g b / alpha)`);
+    // Testing invalid separator
+    test_invalid_value(`color`, `rgb(from rebeccapurple, r, g, b)`);
+    test_invalid_value(`color`, `rgba(from rgb(10%, 20%, 30%, 40%), r, g, b, alpha)`);
```

Comment:
Tests the original intent, that legacy notation isn't valid for relative colors.